packages/lisk-p2p/src/p2p_types.ts

```diff
@@ -21,6 +22,46 @@ export interface IP2PResponsePacket {}
 
 export interface IP2PNodeStatus {}
 
+export interface Logger {
+	readonly error: (message: string) => void;
+	readonly info: (message: string) => void;
+	readonly log: (message: string) => void;
+	readonly trace: (message: string) => void;
+	readonly warn: (message: string) => void;
+}
+
+// ---- Start IP2PConfig interfaces ----
+
+export interface P2PConfigPeersAccess {
+	readonly blacklist: ReadonlyArray<string>;
+}
+
+export interface P2PConfigPeersOptions {
+	readonly broadhashConsensusCalculationInterval: number;
+	readonly timeout: number;
+	readonly wsEngine: string;
+}
+
+export interface P2PConfigPeers {
+	readonly access: P2PConfigPeersAccess;
+	readonly enabled: boolean;
+	readonly list: ReadonlyArray<string>;
+	readonly options: P2PConfigPeersOptions;
+}
+
+export interface P2PConfig {
+	readonly logger?: Logger;
+	readonly peers: P2PConfigPeers;
```

Comment:
`peers` are confusing name for config. maybe options?
Also, important setting should be on the root level